config/delegates.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
Need comment with info about delegates: what is it and why?

Reply:
I guess we need comments overall in each config file.

Reply:
Will add it later.